external-health-monitor/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.8
+
+LABEL maintainers="Kubernetes Authors"
+LABEL description="CSI External Health Monitor Controller"
```

Comment:
`Rebuild of CSI External Health Monitor Controller for Red Hat Marketplace` is more appropriate